### ISSUE TITLE
请问如果训练数据尺寸通常比较小，需不需要调整配置参数中的EastRandomCropData size

### ISSUE THREAD

**Issue by jjkgsd** [PaddlePaddle/PaddleOCR#15069]
### 🔎 Search before asking

- [x] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.
- [x] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.
- [x] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### 🐛 Bug (问题描述)

请问如果训练数据尺寸通常比较小，需不需要调整配置参数中的EastRandomCropData size

![Image](https://github.com/user-attachments/assets/9108706a-9c99-4c79-a8b2-0f08df714b1e)

### 🏃‍♂️ Environment (运行环境)

python 3.8

### 🌰 Minimal Reproducible Example (最小可复现问题的Demo)

python tools/train.py -c configs/train_config.yml

**Comment by fanruifeng** [comment]
请问老哥 修改完尺寸 做微调 检测的效果如何

**Comment by Sunting78** [comment]
您好 建议根据自己的数据进行修改 需为32的倍数

**Comment by fanruifeng** [comment]
> 您好 建议根据自己的数据进行修改 需为32的倍数

您好 我基于自己的数据图片修改成size为 320*320了, 训练出来发现拟合也不好； best metric, hmean: 0.10951008645533142, is_float16: False, precision: 0.1623931623931624, recall: 0.08260869565217391, fps: 21.913784134590703, best_epoch: 4
我基于PPOCRV4的教师模型 做预训练模型,来微调我的数据, 修改size前 和修改后 发现拟合的都不是很好, hmean最高的才达到0.5, 直接用ppocrV4来做检测效果还不错, 微调后效果比较差劲

**Comment by fanruifeng** [comment]
> 您好，这可能也和您的数据量有关，使用多少数据做的微调呢？微调了多少轮次呢？可以放下配置

训练集有2300张数据做微调； 微调了200轮, 大概在100轮左右收敛结束, 最好的human为:0.5;
我的配置如下:
Global:
  debug: false
  use_gpu: true
  epoch_num: &epoch_num 200
  log_smooth_window: 20
  print_batch_step: 100
  save_model_dir: ./output/tag_01
  save_epoch_step: 50
  eval_batch_step:
  - 0
  - 1000
  cal_metric_during_train: false
  checkpoints:
  pretrained_model: pretrain_models/PP-OCRv4_server_det_pretrained.pdparams
  save_inference_dir: null
  use_visualdl: false
  infer_img: doc/imgs_en/img_10.jpg
  save_res_path: ./checkpoints/det_db/predicts_db.txt
  distributed: true

Architecture:
  model_type: det
  algorithm: DB
  Transform: null
  Backbone:
    name: PPHGNet_small
    det: True
  Neck:
    name: LKPAN
    out_channels: 256
    intracl: true
  Head:
    name: PFHeadLocal
    k: 50
    mode: "large"
    fix_nan: True
    

Loss:
  name: DBLoss
  balance_loss: true
  main_loss_type: DiceLoss
  alpha: 5
  beta: 10
  ohem_ratio: 3

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
#    name: Cosine
    name: Const
    learning_rate: 0.0001 #(8*8c)
    warmup_epoch: 2
  regularizer:
    name: L2
    factor: 1e-6

PostProcess:
  name: DBPostProcess
  thresh: 0.2
  box_thresh: 0.5
  max_candidates: 1000
  unclip_ratio: 1.5

Metric:
  name: DetMetric
  main_indicator: hmean

Train:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/tag_train/train/
    label_file_list:
      - ./train_data/tag_train/train.txt
    ratio_list: [1.0]
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - DetLabelEncode: null
    - CopyPaste: null
    - IaaAugment:
        augmenter_args:
        - type: Fliplr
          args:
            p: 0.5
        - type: Affine
          args:
            rotate:
            - -10
            - 10
        - type: Resize
          args:
            size:
            - 0.5
            - 3
    - EastRandomCropData:
        size:
        - 640
        - 640
        max_tries: 50
        keep_ratio: true
    - MakeBorderMap:
        shrink_ratio: 0.4
        thresh_min: 0.3
        thresh_max: 0.7
        total_epoch: *epoch_num
    - MakeShrinkMap:
        shrink_ratio: 0.4
        min_text_size: 8
        total_epoch: *epoch_num
    - NormalizeImage:
        scale: 1./255.
        mean:
        - 0.485
        - 0.456
        - 0.406
        std:
        - 0.229
        - 0.224
        - 0.225
        order: hwc
    - ToCHWImage: null
    - KeepKeys:
        keep_keys:
        - image
        - threshold_map
        - threshold_mask
        - shrink_map
        - shrink_mask
  loader:
    shuffle: true
    drop_last: false
    batch_size_per_card: 8
    num_workers: 2

Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/tag_train/val/
    label_file_list:
      - ./train_data/tag_train/val.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - DetLabelEncode: null
    - DetResizeForTest:
    - NormalizeImage:
        scale: 1./255.
        mean:
        - 0.485
        - 0.456
        - 0.406
        std:
        - 0.229
        - 0.224
        - 0.225
        order: hwc
    - ToCHWImage: null
    - KeepKeys:
        keep_keys:
        - image
        - shape
        - polys
        - ignore_tags
  loader:
    shuffle: false
    drop_last: false
    batch_size_per_card: 1
    num_workers: 2
profiler_options: null

**Comment by Sunting78** [comment]
您好 评估的尺度也修改下吧 Eval: 中的- DetResizeForTest:参数，默认是会把图像短边缩放不小于736 您可以按照您的尺寸修改

**Comment by TingquanGao** [comment]
The issue has no response for a long time and will be closed. You can reopen or new another issue if are still confused.

---
_From Bot_
